arrow-schema/src/schema.rs

```diff
@@ -57,6 +57,17 @@ impl SchemaBuilder {
         self.fields.remove(idx)
     }
 
+    /// Change the FieldRef as index `idx`
+    /// if index out of bounds, will panic
+    pub fn change_field(&mut self, idx: usize, field: impl Into<FieldRef>) {
+        self.fields[idx] = field.into()
```

Comment:
```suggestion
    /// Returns a mutable reference to the field at `idx`
    pub fn field_mut(&mut self, idx: usize) -> &mut FieldRef {
        &mut self.fields[idx]
```
What do you think about this instead?

Reply:
I think it's good!!! 😄